docs/v8-updates.md

```diff
@@ -66,16 +66,19 @@ To update the version of V8 used by workerd, the steps are:
     `strip_prefix` and `url` in the `http_archive` for V8 should be updated based on the new V8
     version/tag.
 
+    The `integrity` check needs to be updated to the new value. You can get the new value in
+    bazel's preferred format just by looking into the mismatch error while trying to compile
+    workerd using the newer V8 version.
+
     See [V8 http_archive in WORKSPACE](https://github.com/cloudflare/workerd/blob/587ad90dd1e91d2660c271018056f4189fca3501/WORKSPACE#L408)
 
 10. Update V8's dependencies in `workerd/WORKSPACE`.
 
-    The `v8/.gclient_entries` file contains the commit versions for V8's dependencies.
-
-    You can find V8's dependencies that are carried through to workerd in the `WORKSPACE` file.
+    You can find the commit versions for V8's dependencies under `v8/.gclient_entries` and the ones
```

Comment:
I know that this was already the case so far/you didn't add this but: I would prefer pointing to V8's `DEPS` here to get the dependency commits since it's part of the repository and tied to a given V8 commit – if someone forgets to run `gclient sync` after changing the V8 commit the `v8/.gclient_entries` entry will be outdated, i.e. may point to the wrong version.

Reply:
Ah, I didn't know. True, this seems to make more sense, although the other file is a little more compact. Updated!